js/client.js

```diff
@@ -1235,7 +1235,7 @@
 						if (target === 'appeal' || target === 'appeals') target = 'view-help-request--appeal';
 						if (target === 'report') target = 'view-help-request--report';
 						if (this.host === 'replay.pokemonshowdown.com') {
-							if (!target || target === 'search') {
+							if (!target || target === 'search' || this.className === 'no-panel-intercept') {
```

Comment:
I would put the check outside of the `this.host === 'replay.pokemonshowdown.com'` check.